dynawo/sources/Simulation/DYNSimulation.cpp

```diff
@@ -401,6 +417,36 @@ Simulation::configureCurveOutputs() {
   }
 }
 
+void
+Simulation::configureFinalStateValueOutputs() {
+  // Final state value settings
+  if (jobEntry_->getOutputsEntry()->getFinalStateValuesEntry()) {
+    string finalStateValuesDir = createAbsolutePath("finalStateValues", outputsDirectory_);
+    if (!is_directory(finalStateValuesDir))
+      create_directory(finalStateValuesDir);
+
+    //---- outputFile ---
+    setFinalStateValuesOutputFile(jobEntry_->getOutputsEntry()->getFinalStateValuesEntry()->getOutputFile());
```

Comment:
I don't understand why we call this twice (also called on line 444)

Reply:
It was copied from curves handling.

In fact, the outputFile attribute in curves is not used. The export is always made to a predefined filename. The xsd for job does not consider an attribute outputFile. The Xml handler for curves entry in jobs only reads attributes inputFile and exportMode.

I will make a change to use the output file name if it is given in the entry. If it is empty, then use a default name.

Reply:
... but that will introduce noise on handling both curves and final state values output files
should be they interpreted as absolute? relative to outputs? ...

probably is better (at least in this PR) to simply ignore the `outputFile` (with an explicit comment about it).

Reply:
I decided to remove the `outputFile` attribute from `FSV`, leaving only the last call to `setFinalStateValuesOutputFile` with a hard-coded value, the same way it is done with curves.